docs/modules/plugins/pages/plugin-datetime.adoc

```diff
@@ -306,34 +307,69 @@ Converts the input date to the https://en.wikipedia.org/wiki/Unix_time[Unix epoc
 * *`$inputDate`* - the date to be converted in the {iso-date-format-link} format.
 
 [cols="2,>1", options="header"]
-.Examples of the expressions converting the input date to the Unix epoch time
+.Examples of the expressions converting the input date to the Unix epoch time in seconds
 |===
 |Expression
 |Result
 
 |`#{toEpochSecond(1993-04-16T00:00:00)}`
 |734918400
 
+|`#{toEpochSecond(1993-04-16T00:00:00.123)}`
+|734918400
+
 |`#{toEpochSecond(2020-12-11T18:43:05+05:30)}`
 |1607692385
 
+|`#{toEpochSecond(2020-12-11T18:43:05.987+05:30)}`
+|1607692385
+|===
+
+=== `toEpochMilli`
```

Comment:
I believe it should be millis, not milli, toEpochSecond also seems to be wrong

Reply:
but we don't expose java method names to users, so from user point of view millis/seconds look more relevant

Reply:
> but we don't expose java method names to users

true, but I was talking about naming, and my guess is Java naming is correct from English language standpoint

> so from user point of view millis/seconds look more relevant

I'm not native speaker, thus I can rely only on external sources (like, Java)